build.sbt

```diff
@@ -21,20 +22,30 @@ resolvers += "softprops-maven" at "http://dl.bintray.com/content/softprops/maven
 val akkaStreamV      = "2.0.3"
 val scalaTestV       = "3.0.0-M15"
 val scalaMockV       = "3.2.2"
+val slickPgV         = "0.11.2"
+val slickV           = "3.1.1"
 
 libraryDependencies ++= List(
-  "com.typesafe.slick"      %% "slick"                                % "3.0.1",
+  "org.scala-lang.modules"  %% "scala-xml"                            % "1.0.4",
+  
+  "com.typesafe.slick"      %% "slick"                                % slickV,
+  "com.typesafe.slick"      %% "slick-hikaricp"                       % slickV exclude("com.zaxxer", "HikariCP-java6"),
+
   "com.typesafe.akka"       %% "akka-stream-experimental"             % akkaStreamV,
   "com.typesafe.akka"       %% "akka-http-core-experimental"          % akkaStreamV,
   "com.typesafe.akka"       %% "akka-http-spray-json-experimental"    % akkaStreamV,
+
   "com.typesafe.akka"       %% "akka-slf4j"                           % "2.3.9",
   "ch.qos.logback"           % "logback-classic"                      % "1.1.3",
+
   "com.google.inject"        % "guice"                                % "4.0",
   "net.codingwell"          %% "scala-guice"                          % "4.0.0",
-  "org.postgresql"           % "postgresql"                           % "9.3-1103-jdbc41",
-  "com.github.tminglei"     %% "slick-pg"                             % "0.9.1",
-  "com.zaxxer"               % "HikariCP"                             % "2.4.1",
+  "org.postgresql"           % "postgresql"                           % "9.4-1206-jdbc42",
+  "com.github.tminglei"     %% "slick-pg"                             % slickPgV,
+  "com.github.tminglei"     %% "slick-pg_date2"                       % slickPgV,
+  "com.zaxxer"               % "HikariCP"                             % "2.4.3",
```

Comment:
Right. As I remember before they had java8 version and for java6/java7 there was a separate version with "-java6" suffix. Looks like that they changed it.